yarn.lock

```diff
@@ -16592,27 +17125,10 @@ write-file-atomic@^4.0.2:
     imurmurhash "^0.1.4"
     signal-exit "^3.0.7"
 
-ws@^6.1.0:
-  version "6.2.2"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-6.2.2.tgz#dd5cdbd57a9979916097652d78f1cc5faea0c32e"
-  integrity sha512-zmhltoSR8u1cnDsD43TX59mzoMZsLKqUweyYBAIvTngR3shc0W6aOZylZmq/7hqyVxPdi+5Ud2QInblgyE72fw==
-  dependencies:
-    async-limiter "~1.0.0"
-
-ws@^7.3.1:
-  version "7.5.9"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-7.5.9.tgz#54fa7db29f4c7cec68b1ddd3a89de099942bb591"
-  integrity sha512-F+P9Jil7UiSKSkppIiD94dN07AwvFixvLIj1Og1Rl9GGMuNipJnV9JzjD6XuqmAeiswGvUmNLjr5cFuXwNS77Q==
-
-ws@^8.11.0:
-  version "8.14.2"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-8.14.2.tgz#6c249a806eb2db7a20d26d51e7709eab7b2e6c7f"
-  integrity sha512-wEBG1ftX4jcglPxgFCMJmZ2PLtSbJ2Peg6TmpJFTbe9GZYOQCDPdMYu/Tm0/bGZkw8paZnJY45J4K2PZrLYq8g==
-
-ws@^8.13.0, ws@^8.2.3:
-  version "8.13.0"
-  resolved "https://registry.yarnpkg.com/ws/-/ws-8.13.0.tgz#9a9fb92f93cf41512a0735c8f4dd09b8a1211cd0"
-  integrity sha512-x9vcZYTrFPC7aSIbj7sRCYo7L/Xb8Iy+pW0ng0wt2vCJv7M9HOMy0UoN3rr+IFC7hb7vXoqS+P9ktyLLLhO+LA==
+ws@>=7.5.10, ws@>=8.17.1, ws@^6.1.0, ws@^7.3.1, ws@^8.11.0, ws@^8.13.0, ws@^8.2.3:
+  version "8.17.1"
```

Comment:
Maybe I shouldn't be worrying too much about yarn internals but I'm confused about how `8.17.1` satisfies all the versions listed above. E.g. `ws@^6.1.0` doesn't seem like it should be satisfied with `8.17.1`.

Reply:
It's a good point, I may have been too greedy with the globbing in the `resolutions` block in package.json .. I'll see if I can be more specific where multiple peer dependency versions are used

Reply:
Ah right sorry I hadn't quite appreciated that change in the `resolutions` block for some reason! Yeah could be worth being more specific (it looks like both v7 and v8, which are affected, have a patch available). So it might be just a case of making sure any v7 or v8 in the project are on the latest version of that major release. Having said all that, if it works then 👍